tests/boostrap.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of Transfer.
+ *
+ * For the full copyright and license information, please view the LICENSE file located
+ * in the root directory.
+ */
+$loader = require __DIR__.'/../vendor/autoload.php';
+
+$configDir = __DIR__.'/../vendor/ezsystems/ezpublish-kernel';
+
+if (!file_exists($configDir.'/config.php')) {
+    if (!symlink($configDir.'/config.php-DEVELOPMENT', $configDir.'/config.php')) {
+        throw new \RuntimeException('Could not symlink config.php-DEVELOPMENT to config.php!');
+    }
+}
+
+$loader->addPsr4('Transfer\\EzPlatform\\Tests\\', __DIR__.'/../../ezplatform/src/Transfer/EzPlatform');
+$loader->addPsr4('Transfer\\EzPlatform\\Tests\\', __DIR__.'/../../ezplatform/tests/Transfer/EzPlatform/Tests');
```

Comment:
@valisj 
In YamlToEzPlatformContentTypeTestCase I'm using EzPlatformTestCase which kind of messes up/duplicates this bootstrap-file with the boostrap in transfer/ezplatform. Is this ok, or do you have a better suggestion? I don't really want to recreate another EzPlatformTestCase just because of the two loads on L19 and L20. Thoughts?